src/main/java/Deadline.java

```diff
@@ -0,0 +1,23 @@
+public class Deadline extends Task{
+    protected String deadline;
+
+
+    public Deadline(String description, String deadline) {
+        super(description);
+        this.deadline=deadline;
+        String echo_msg ="          ____________________________________________________________\n" +
+                "          Got it. I've added this task:\n"+
+                "          "+printTask()+"\n"+
+                "          Now you have "+ Task.taskCounter +" in the list.\n"+
+                "          ____________________________________________________________";
+
+
+        System.out.println(echo_msg);
+    }
```

Comment:
I think it would be clearer to create another method for printing the text and then call from main instead of putting it in the constructor.